{{ cookiecutter.package_name|slugify }}/src/{{ cookiecutter.package_name|slugify|replace("-", "_") }}/__init__.py

```diff
@@ -1 +1,29 @@
 """{{ cookiecutter.package_name }} package."""
+{%- if cookiecutter.with_fastapi_api|int or cookiecutter.with_streamlit_app|int %}
+
+import logging
+
+import coloredlogs
+
+
+def configure_root_logger() -> None:
+    """Configure the root logger."""
+    # Remove all handlers associated with the root logger object.
+    for handler in logging.root.handlers:
+        logging.root.removeHandler(handler)
+    # Add coloredlogs' coloured StreamHandler to the root logger.
+    coloredlogs.install(
+        fmt="%(asctime)s %(name)s %(funcName)s %(levelname)s %(message)s",
+        datefmt="%H:%M:%S",
+        field_styles={
+            "asctime": {"color": "green"},
+            "name": {"color": "blue"},
+            "funcName": {"color": "magenta"},
+            "levelname": {"color": "black", "bold": True},
+        },
+        isatty=True,
+    )
+
+
+configure_root_logger()
+{%- endif %}
```

Comment:
Not sure if it's because of the way that GitHub renders files in the browser, but we need a newline after this `{%- endif %}` in order to satisfy the [end-of-file-fixer](https://github.com/pre-commit/pre-commit-hooks#end-of-file-fixer) linter.

Reply:
Yes, its just the Github rendering. There is actually a new line :)